.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: Release Charts
+
+on: 
+  workflow_dispatch:
+    inputs:
+      csi_operator_version:
+        description: 'Last released CSI Operator version'
+        required: true
+      release_tag:
+        description: 'Current release tag'
+        required: true
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      
+      - name: Make version
+        id: vars
+        run: |
+          echo "::set-output name=version::$(make version)"
+          echo "::set-output name=commit_hash::$(git rev-parse --short HEAD)"
+
+      - name: Create asset
+        run: |
+          PACKAGE_NAME=csi-baremetal-deployment-${{ github.event.inputs.csi_operator_version }}.tgz
+          DIR=/tmp/csi-baremetal-deployment-${{ steps.vars.outputs.version }}
+          cp -r ./charts $DIR
```

Comment:
It's better to use `artifacts` or `cache` actions here. There is no guarantee that files in `/tmp` folder will be persist between jobs.